.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: node_js
 node_js:
-  - 8
   - 10
   - 12
+  - 13
```

Comment:
I don't see the point in adding Node.js 13 here since it was about to reach EOL when this PR was opened and it had reached it already when it was landed.